src/config_file.rs

```diff
@@ -117,6 +117,12 @@ impl Config {
         serde_yaml::from_str::<Self>(yaml)
     }
 
+    #[cfg(not(windows))]
```

Comment:
Don't think you will have to implement this. The reason why we have the other functions as well the `#[cfg()]` blocks was so that we can figure out the default directory to use on each system.